src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -0,0 +1,166 @@
+  
+      <MonoAOTCompilerDefaultAotArguments Include="static" />
+      <MonoAOTCompilerDefaultAotArguments Include="dwarfdebug" />
+
+      <MonoAOTCompilerDefaultAotArguments Include="nimt-trampolines=2000" />
+      <MonoAOTCompilerDefaultAotArguments Include="ntrampolines=10000" />
+      <MonoAOTCompilerDefaultAotArguments Include="nrgctx-fetch-trampolines=256" />
+      <MonoAOTCompilerDefaultAotArguments Include="ngsharedvt-trampolines=4400" />
+      <MonoAOTCompilerDefaultAotArguments Include="nftnptr-arg-trampolines=4000" />
+      <MonoAOTCompilerDefaultAotArguments Include="nrgctx-trampolines=31000" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <AotArguments>@(MonoAOTCompilerDefaultAotArguments, ';')</AotArguments>
+      <ProcessArguments>@(MonoAOTCompilerDefaultProcessArguments, ';')</ProcessArguments>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_AotInputAssemblies Include="@(_AndroidAssembliesInternal)" 
+                           Condition="'%(_AndroidAssembliesInternal._InternalForceInterpret)' != 'true'">
+        <AotArguments>$(AotArguments)</AotArguments>
+        <ProcessArguments>$(ProcessArguments)</ProcessArguments>
+      </_AotInputAssemblies>
+
+      <_AOT_InternalForceInterpretAssemblies Include="@(_AndroidAssembliesInternal->WithMetadataValue('_InternalForceInterpret', 'true'))" />
+      <_AndroidAssembliesInternal Remove="@(_AndroidAssembliesInternal)" />
+    </ItemGroup>
+    
+    <MakeDir Directories="$(_MobileIntermediateOutputPath)" />
+  </Target>
+
+  <Target Name="_AndroidPrepareProfiledAot"
+          Condition="'$(NetTracePath)' != '' and '$(ForceFullAOT)' != 'true'"
+          DependsOnTargets="_AndroidBeforeAotCompileApp">
+    <PropertyGroup>
+      <_ToolPath>$([System.IO.Path]::GetDirectoryName('$(DotnetPgoToolPath)'))</_ToolPath>
+      <_ToolExe>$([System.IO.Path]::GetFileName('$(DotnetPgoToolPath)'))</_ToolExe>
+    </PropertyGroup>
+
+    <NetTraceToMibcConverter
+      ToolPath="$(_ToolPath)"
+      ToolExe="$(_ToolExe)"
+      Assemblies="@(_AotInputAssemblies)"
+      NetTracePath="$(NetTracePath)"
+      OutputDir="$(_MobileIntermediateOutputPath)">
+      <Output TaskParameter="MibcProfilePath" PropertyName="_MibcProfilePath" />
+    </NetTraceToMibcConverter>
+  </Target>
+
+  <Target Name="_AndroidAotCompileApp"
+          Condition="'$(RunAOTCompilation)' == 'true'"
+          DependsOnTargets="_AndroidBeforeAotCompileApp">
+
+    <MonoAOTCompiler
+        CompilerBinaryPath="@(MonoAotCrossCompiler->WithMetadataValue('RuntimeIdentifier','$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())'))"
+        OutputDir="$(_MobileIntermediateOutputPath)"
+        Mode="$(_AOTMode)"
+        OutputType="AsmOnly"
+        Assemblies="@(_AotInputAssemblies)"
+        AotModulesTablePath="$(AndroidAppBundleDir)\modules.c"
+        IntermediateOutputPath="$(_MobileIntermediateOutputPath)"
+        MibcProfilePath="$(_MibcProfilePath)"
```

Comment:
Previously, we allowed `MibcProfilePath` to be specifiable by the user. It looks like we are hiding this now (the fact that it has a leading `_`) and it will only be set if NetTraceToMibcConverter is invoked (or the task overrides any preset `_MibcProfilePath`)

Perhaps we can have something like 

```
    <Output TaskParameter="MibcProfilePath" PropertyName="_NetTraceGeneratedMibcProfilePath" />
  </NetTraceToMibcConverter>
</Target>

<Target Name="_AndroidAotCompileApp" ...

  <PropertyGroup>
    <_MibcProfilePath>$(MibcProfilePath);$(_NetTraceGeneratedMibcProfilePath)</_MibcProfilePath>
  </PropertyGroup>

<MonoAOTCompiler 
    ...
```

Reply:
Good catch. I forgot to switch this over to items.